client/platform/desktop/frontend/components/Recent.vue

```diff
@@ -86,18 +88,28 @@ export default defineComponent({
       }
     }
 
-    const filteredRecents = computed(() => recents.value
-      .filter((v) => v.name.toLowerCase().indexOf((searchText.value || '').toLowerCase()) >= 0));
-    const paginatedRecents = computed(() => (filteredRecents.value.slice(0, limit.value)));
-    const totalRecents = computed(() => filteredRecents.value.length);
-
-    function toggleMore() {
-      if (limit.value < recents.value.length) {
-        limit.value = recents.value.length;
-      } else {
-        limit.value = pageSize;
+    async function confirmDeleteDataset(datasetId: string, datasetName: string) {
+      const result = await prompt({
+        title: 'Confirm',
+        text: `Do you want to delete dataset ${datasetName}`,
```

Comment:
Please include very scary warning about exactly what this will remove.

1) It will not remove source media, such as images or video.
2) It will not remove annotation files that were imported when the dataset was created.
3) It will completely remove any annotations that have been created in DIVE.  

Maybe even mention that you can use the export button to create a backup before deletion.